.travis.yml

```diff
@@ -1,5 +1,14 @@
+# According to https://docs.travis-ci.com/user/ci-environment,
+# `sudo: required` must be set to use `dist: trusty`
+dist: trusty
```

Comment:
The TravisCI default is 12.04, which has some ABI incompatibilities with native bindings to things like libuuid